config/config_tests.xml

```diff
@@ -244,7 +244,12 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <BFBFLAG>TRUE</BFBFLAG>
     <STOP_OPTION>ndays</STOP_OPTION>
     <STOP_N>11</STOP_N>
+    <REST_N>$STOP_N / 2 + 1</REST_N>
```

Comment:
These were being set manually in the python. There's no reason to do that since it can be done in the XML. The tweaks needed for the second run of erp are still being done in case_two_setup.